packages/invoices/src/InboundShipment/api/hooks.ts

```diff
@@ -1,7 +1,255 @@
-import { useOmSupplyApi } from '@openmsupply-client/common';
+import { useCallback } from 'react';
```

Comment:
these have also been copy pasted from where they lived, renamed and changed slightly to use the `api` object and `invoiceNumber` rather than `id`